.github/workflows/build-and-test.yml

```diff
@@ -187,10 +177,11 @@ jobs:
       run: |
         source env/activate
         export SYSTEM_DESC_PATH="${GITHUB_WORKSPACE}/ttrt-artifacts/system_desc.ttsys"
-        cmake --build build -- check-ttmlir
+        ln -sf ${{ steps.strings.outputs.install-output-dir }} ${{ steps.strings.outputs.build-output-dir }}
```

Comment:
This is kind of the only hacky thing, use a link to make the install dir look like a phony build dir since all of the paths in the artifact expect the build dir.